packages/pnpm-sync/src/start.ts

```diff
@@ -58,7 +65,23 @@ program
           if (lockfile === null) {
             return undefined;
           } else {
-            return lockfile;
+            const lockfilePackages: Record<string, ILockfilePackage> = {};
+            for (const packagePath in lockfile.packages) {
```

Comment:
What ESLint ruleset are you using in this project?  

`for..in` is usually deprecated by lint rules; the safe syntax is `for (const packagePath of Object.keys(lockfile.packages))`.